ui/app/css/itcss/components/account-dropdown.scss

```diff
@@ -1,13 +1,16 @@
 .account-dropdown-name {
-  
+  font-family: 'DIN OT';
 }
 
 .account-dropdown-balance {
   color: $dusty-gray;
+  line-height: 19px;
 }
 
 .account-dropdown-edit-button {
   color: $dusty-gray;
+  font-family: "DIN OT";
+  font-weight: 500;
```

Comment:
I am not sure that this `font-weight` will affect anything, given the way fonts work in this app. You might need to add "DIN OT Bold" as a new font and use that instead.